src/ralph/api/routers/statements.py

```diff
@@ -251,7 +251,7 @@ async def get(
         response.update(
             {
                 "more": (
-                    f"{path}{query + '&' if query else '?'}pit_id={query_result.pit_id}"
+                    f"{path}?{query + '&' if query else ''}pit_id={query_result.pit_id}"
```

Comment:
Maybe it means that we need to build the `more` URL with "more" reliable tools such as `urllib` [1]. WDYT?

[1] https://docs.python.org/3/library/urllib.parse.html